index.html

```diff
@@ -127,6 +127,14 @@ <h6 class="header3">About Myself</h6>
                 </nav>
                  
             </section>
+            <footer class="footer">
+                <h2 class="contact">Contact me</h2>
+                <p class="info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <input type="text" name="name" maxlength="30" placeholder="Iptisam Dawood">
+                <input type="email" name="email" placeholder="mukadamiptisam@gmail.com">
+                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+                <button class="btn" type="submit">Get it Touch</button>
+            </footer>
```

Comment:
- [ ] All input fields need to be wrapped in a single `form` element.
- [ ] Form must use the action and method attribute to integrate with [formspree service](https://formspree.io/html).
- `action` value should be similar to `https://formspree.io/f/{form_id}`
- `method` value equals `POST`
- [ ] All input fields have the `required` attribute to activate the validation on submit.
- [ ] Please use flexbox to layout the elements.